drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/PatternPage.java

```diff
@@ -107,6 +109,10 @@ void setupPattern() {
 
         setUpPatternList();
 
+        if (!isPatternCreationEnabled()) {
```

Comment:
I don't think so. I'm disabling the pattern creation only when the wizard is editing a "ActionSetFieldCol52" or a "ActionWorkItemSetFieldCol52" :-)